packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/nodejs/sdkforyourapp.md

```diff
@@ -1,41 +1,12 @@
-## TO DO for NodeJS
 
-#### Step 1: Set up your app for .Net 4.8 or greater
+#### Step 1: Install the SDK into your project application
 
-The SDK and samples are built using .Net 4.8.
+Download and install the SDK into your Express application. See [Building the SDK](https://github.com/okta/okta-auth-js#building-the-sdk) for further information.
 
-#### Step 2: Add the Okta SDK Nuget Packages
+#### Step 2: Install the module dependencies
```

Comment:
step 1 & 2 can be combined into one step

Reply:
As mentioned, I recast this section.